### PR TITLE
928338 - Added Customization video link for Autocomplete, Combobox and Carousel [Hotfix].

### PR DESCRIPTION
**Pull request by ManjudhanasekaranSF4622**
Added Customization video link for Autocomplete, Combobox and Carousel controls.

### CODE REVIEW

**Comment by PrithisIyyappan** [comment]
Hi @Choza-rajan, I've reviewed the PR. Please ensure and merge the same.

File: MAUI/Autocomplete/UI-Customization.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ documentation: ug
 
 This section explains different UI customizations available in the [.NET MAUI Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) control.
 
+Watch this video to quickly customize the .NET MAUI Autocomplete.
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/UI-Customization.md:14]: Watch this video for a quick guide on customizing the .NET MAUI Autocomplete.

Comment (ManjudhanasekaranSF4622) [reply MAUI/Autocomplete/UI-Customization.md]: Content modified

File: MAUI/Carousel-View/images/LoadMore.png
Comment (PrithisIyyappan) [inline MAUI/Carousel-View/images/LoadMore.png:1]: Why the size of the image is varied?

Comment (ManjudhanasekaranSF4622) [reply MAUI/Carousel-View/images/LoadMore.png:1]: While compressing the image with original size image quality is not okay. Now I have changed the image with original size.

File: MAUI/Autocomplete/UI-Customization.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ documentation: ug
 
 This section explains different UI customizations available in the [.NET MAUI Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) control.
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/UI-Customization.md:13]: @ManjudhanasekaranSF4622 Can you add the task id?

Comment (ManjudhanasekaranSF4622) [reply MAUI/Autocomplete/UI-Customization.md:13]: yes sir.

File: MAUI/Autocomplete/UI-Customization.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ documentation: ug
...
 
+Watch this video for a quick guide on customizing the .NET MAUI Autocomplete.
+
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/UI-Customization.md:15]: @ManjudhanasekaranSF4622 , To get start quickly to customize the Appearance of .NET MAUI Autocomplete, you can check on this video:

Comment (ManjudhanasekaranSF4622) [reply MAUI/Autocomplete/UI-Customization.md:15]: Sir, content revamped.

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**